mne/decoding/search_light.py

```diff
@@ -350,7 +362,9 @@ def _sl_transform(estimators, X, method, pb):
         if ii == 0:
             y_pred = _sl_init_pred(_y_pred, X)
         y_pred[:, ii, ...] = _y_pred
-        pb.update(ii + 1)
+
+        if pb is not None:
```

Comment:
Same

Reply:
oops, I should have removed these, thanks!